CONTRIBUTING.md

```diff
@@ -1,5 +1,63 @@
-## Contributing
+# Contributing
 
 If you would like to contribute please read OpenTelemetry Collector
 [contributing guidelines](https://github.com/open-telemetry/opentelemetry-collector/blob/master/CONTRIBUTING.md)
 before you begin your work.
+
+## Adding New Components
+Before you start please read the [contributing
+guidelines](https://github.com/open-telemetry/opentelemetry-collector/blob/master/CONTRIBUTING.md).
```

Comment:
Duplicate information. We already have this a few lines above. Eliminate one of the paragraphs.

Reply:
done.